modules/data.land/R/helpers.R

```diff
@@ -0,0 +1,7 @@
+stop <- function(...) {
+    if (requireNamespace(PEcAn.utils)) {
+        PEcAn.utils::logger.error(...)
+    } else {
+        stop(...)
```

Comment:
doesn't this have to be base::stop so that you don't just get an infinite recursion? Also, are you sure you want to map stop to logger.error rather than logger.severe, since logger.error doesn't call base::stop?